### ISSUE TITLE
Compatibility issue with Mo Creatures, please help?

### ISSUE THREAD

**Issue by Skylandy** [Tschipp/CarryOn#130]
### Expected Behavior
Wild mobs work perfectly, from what I've seen, with most mods I'm using. The only problem seems to be with Mo creatures, I approach a tamed mob (wild are fine) and try to pick it up...

### Actual Behavior
Server: Immediate crash on player side. Cannot confirm at time if other players can still join, but player cannot rejoin post crash. Restarting the server doesn't help.

Singleplayer: This doesn't actually crash straight away. Instead a strange behaviour can be observed whereby the tamed mob remains in place whilst a duplicate of the mob is picked up. If the player tries to place the mob back on the ground, the 'clone' vanishes.

### Steps to Reproduce
Install MoCreatures 1.12.2 and CarryOn (custom spawner and a variety of other mods included, but all seem to interact fine besides MoCreatures).

I've also restarted the singleplayer world and the issue persists.

### Version of Minecraft, Carry On, Forge
1.12.2 - MC version
forge-1.12.2-14.23.4.2705
Carry On - 1.12.2 v1.10 (latest as of posting)

### Screenshots encouraged
![2018-08-16_23 23 52](https://user-images.githubusercontent.com/35546233/44238629-7807ee80-a1ad-11e8-8f74-4140bbdea05f.png)
(Image above of testing on singleplayer superflat world. The lion held is the lion named 'Bu' and yet you can see it is only a duplicate.)

**Comment by Tschipp** [comment]
Very weird. Will take a look at it

**Comment by Neeve01** [comment]
Can confirm. Crashes everyone around the erroring mob until deleted from NBT directly.

https://hastebin.com/umakeruzon.log

**Comment by Tschipp** [comment]
Yes, I will look at it when I have time, in one week.

**Comment by Tschipp** [comment]
I just took a look at it, and i don't notice anything strange. I also tried picking up lions. Does it occur with any other mobs?

**Comment by Skylandy** [comment]
In regards to recreating, were these lions/mobs tamed upon testing? Wild mobs react fine, it is only once they've been tamed (i.e with the medallion and meat for big cats). I've been blacklisting all tamable mobs from the mod, but truly they work fine until they've been tamed/linked to a player.

- On an additional note, temporarily removing carry on from the server and restarting stops the crashes (not sure if tamed mob is deleted, but I don't think so) and can be re-added once the server is running again. I've been using this as a temporary fix alongside the blacklisting.
